utils/rpms/daos.spec

```diff
@@ -241,16 +241,32 @@ install -m 644 utils/systemd/daos-agent.service %{?buildroot}/%{_unitdir}
 %{_libdir}/libduns.so
 %{_libdir}/libdfuse.so
 %{_libdir}/libioil.so
-%{_libdir}/python2.7/site-packages/pydaos_shim_27.so
-%{_libdir}/python2.7/site-packages/pydaos.py
-%{_libdir}/python2.7/site-packages/pydaos.pyc
-%{_libdir}/python2.7/site-packages/pydaos.pyo
+%dir  %{_libdir}/python2.7/site-packages/pydaos
```

Comment:
At some point, we probably want to separate this all out into a daos-python (or somesuch, following existing standards, etc.) sub-package.